nixos/modules/services/x11/xautolock.nix

```diff
@@ -26,9 +26,9 @@ in
         };
 
         locker = mkOption {
-          default = "xlock"; # default according to `man xautolock`
-          example = "i3lock -i /path/to/img";
-          type = types.string;
+          default = "${pkgs.xlockmore}/bin/xlock"; # default according to `man xautolock`
+          example = "${pkgs.i3lock}/bin/i3lock -i /path/to/img";
```

Comment:
Using `literalExample` would make sense here. Otherwise it will put the store path into the manuel.

Reply:
@Mic92 when I tested this, the expression did not get expanded to a store path in the manpages. (also see #34469)

Reply:
This is from my `man configuration.nix`
```
services.xserver.xautolock.locker
           The script to use when automatically locking the computer.

           Type: string

           Default: "\${pkgs.xlockmore}/bin/xlock"

           Example: "\${pkgs.i3lock}/bin/i3lock -i /path/to/img"

           Declared by:
               <nixpkgs/nixos/modules/services/x11/xautolock.nix>
```
(note the escaped '$', that's a part of what the above mentioned PR is about)

Reply:
Ok. The store path does not get expanded. However `literalExample` looks better because it does not add escaping.

Reply:
minor detail though:

```diff
4ed723c622c6d29d6094b25e50c9a20e79a403db nixos/xautolock: remove escaping from examples
diff --git a/nixos/modules/services/x11/xautolock.nix b/nixos/modules/services/x11/xautolock.nix
index a614559970e..23b1eb1607b 100644
--- a/nixos/modules/services/x11/xautolock.nix
+++ b/nixos/modules/services/x11/xautolock.nix
@@ -27,7 +27,9 @@ in
 
         locker = mkOption {
           default = "${pkgs.xlockmore}/bin/xlock"; # default according to `man xautolock`
-          example = "${pkgs.i3lock}/bin/i3lock -i /path/to/img";
+          example = literalExample ''
+            "''${pkgs.i3lock}/bin/i3lock -i /path/to/img"
+          '';
           type = types.str;
 
           description = ''
@@ -37,7 +39,9 @@ in
 
         nowlocker = mkOption {
           default = null;
-          example = "${pkgs.i3lock}/bin/i3lock -i /path/to/img";
+          example = literalExample ''
+            "''${pkgs.i3lock}/bin/i3lock -i /path/to/img"
+          '';
           type = types.nullOr types.str;
 
           description = ''
@@ -56,7 +60,9 @@ in
 
         notifier = mkOption {
           default = null;
-          example = "${pkgs.libnotify}/bin/notify-send \"Locking in 10 seconds\"";
+          example = literalExample ''
+            "''${pkgs.libnotify}/bin/notify-send \"Locking in 10 seconds\""
+          '';
           type = types.nullOr types.str;
 
           description = ''
@@ -66,7 +72,9 @@ in
 
         killer = mkOption {
           default = null; # default according to `man xautolock` is none
-          example = "${pkgs.systemd}/bin/systemctl suspend";
+          example = literalExample ''
+            "''${pkgs.systemd}/bin/systemctl suspend"
+          '';
           type = types.nullOr types.str;
 
           description = ''
```